Assets/Scripts/SS3D/Systems/Tile/PlacedObjects/PlacedItemObject.cs

```diff
@@ -0,0 +1,102 @@
+using FishNet;
+using FishNet.Object;
+using SS3D.Core;
+using SS3D.Logging;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SS3D.Systems.Tile
+{
+    public class PlacedItemObject : NetworkBehaviour
+    {
+        /// <summary>
+        /// Creates a new PlacedItemObject from a prefab at a given position and rotation. Uses NetworkServer.Spawn() if a server is running.
+        /// </summary>
+        /// <param name="worldPosition"></param>
+        /// <param name="origin"></param>
+        /// <param name="rotation"></param>
+        /// <param name="itemSo"></param>
+        /// <returns></returns>
+        public static PlacedItemObject Create(Vector3 worldPosition, Quaternion rotation, ItemObjectSo itemSo)
+        {
+            GameObject placedGameObject = Instantiate(itemSo.prefab);
+            placedGameObject.transform.SetPositionAndRotation(worldPosition, rotation);
+
+            PlacedItemObject placedObject = placedGameObject.GetComponent<PlacedItemObject>();
+            if (placedObject == null)
+            {
+                placedObject = placedGameObject.AddComponent<PlacedItemObject>();
+            }
+
+            placedObject.Setup(worldPosition, rotation, itemSo);
+
+            if (InstanceFinder.ServerManager != null && placedObject.GetComponent<NetworkObject>() != null)
+            {
+                if (placedObject.GetComponent<NetworkObject>() == null)
+                    Punpun.Yell(SystemLocator.Get<TileSystem>(), $"{placedObject.GetNameString()} does not have a Network Component and will not be spawned");
+                else
+                    InstanceFinder.ServerManager.Spawn(placedGameObject);
+            }
+
+            return placedObject;
+        }
+
+
+        /// <summary>
+        /// SaveObject that contains all information required to reconstruct the object.
+        /// </summary>
+        [Serializable]
+        public class PlacedSaveObject
```

Comment:
make that private, or if it has to be accessed from other classes, put it in it's own file.

Reply:
This is more of a design choice to keep the "save" object in the same class as it makes a save of. It has to be public since the tilemap or chunk needs to know it's contents during loading. Let me know if you disagree.

Reply:
I understand the intent, and it's a good one. However this goes against our C# style guide, one declaration per file for class structs, enums .... Could you rename it in SavedPlacedItemObject, make it a struct (for performance reasons), put it in its own file ? 

Do the same for ChunkSaveObject, rename it to SavedTileChunk and in it's own file (don't make this one a struct, its memory footprint is too large.)

If you disagree, I would understand as your choice makes sense too. In this case we'll ask John to make a choice.

Reply:
Moved all save objects to separate files and changed saves for placed objects into structs